src/lib/geolocation.test.ts

```diff
@@ -0,0 +1,29 @@
+import { getCountryName, getLocalCurrencySymbol } from './geolocation';
+
+describe('getLocalCurrencySymbol', () => {
```

Comment:
What about table-driven tests for all these cases, to replace them all with a single test? You get slightly worse error reporting it's true, but it's more concise and easier to add/remove cases. https://dave.cheney.net/2013/06/09/writing-table-driven-tests-in-go

Reply:
Cool yeah I'll give this a go. I've kind of shied away from table driven tests lately (preferring meaningful error messages and more descriptive scenario names) but I think this could be a case where they work well. I'll open a PR and we can see if we like it :-)